src/workloads/query/multiplanner/MultikeyIndexes.yml

```diff
@@ -7,10 +7,6 @@ Description: |
   multi-planner will behave more optimally than the SBE multiplanner because it will cut off execution
```

Comment:
[optional] We are still comparing the classic multi-planner to the SBE multi-planner here. Could consider cleaning that up.